file-handling-example/connector.json

```diff
@@ -0,0 +1,12 @@
+{
+	"name": "file-handling-example",
+	"version": "1.0",
+	"title": "File handling example",
+	"description": "",
+	"service": {
+		"name": "json-placeholder",
+		"version": "1"
+	},
```

Comment:
do we need to change this name to match the name of the connector?
is this field related to service at tray?